build-system/test-configs/forbidden-terms.js

```diff
@@ -1315,7 +1321,7 @@ function stripComments(contents) {
   });
   // Single line comments either on its own line or following a space,
   // semi-colon, or closing brace
-  return contents.replace(/( |}|;|^) *\/\/.*/g, '$1');
+  return contents.replace(/( |}|;|^) *\/\/.*/gm, '$1');
```

Comment:
This needs the multiline flag `m` so that lines starting with an inline comment `^// ...` are matched.